src/requests/getForecast.js

```diff
@@ -0,0 +1,40 @@
+import axios from "axios";
+
+const getForecast = async (
+  setSelectedDate,
+  setForecasts,
+  setLocation,
+  searchText,
+  setErrorMessage
+) => {
+  let endpoint = "https://cmd-shift-weather-app.onrender.com/forecast";
+
+  if (searchText) {
+    setErrorMessage("");
+    endpoint += `?city=${searchText}`;
+  }
+
+  return axios
+    .get(endpoint)
+    .then((response) => {
+      setSelectedDate(response.data.forecasts[0].date);
+      setForecasts(response.data.forecasts);
+      setLocation(response.data.location);
+      setErrorMessage("");
+    })
+    .catch((error) => {
+      const { status } = error.response;
+      if (status === 404) {
+        setErrorMessage("No such town or city, try again!");
+        /* eslint-disable-next-line no-console */
+        console.error("Location is not valid", error);
```

Comment:
Since you are already displaying an error message to the user explaining why things are not working, these console logs could be removed
You could also remove the eslint no console exception above them as a result of this too